src/sentry/static/sentry/app/components/events/sdk.jsx

```diff
@@ -31,7 +31,7 @@ const EventSdk = React.createClass({
               <td className="key">Version</td>
               <td className="value"><pre>{data.version}</pre></td>
             </tr>
-            <tr key="version">
+            <tr key="clientIP">
```

Comment:
this was causing the client ip section to never show up...
![screenshot 2016-11-10 16 30 51](https://cloud.githubusercontent.com/assets/5026776/20200104/e3ccad68-a763-11e6-8752-35f721287749.png)